### PR TITLE
[Makefile/slave docker] ARM arch doesn't support few packages

### PR DESCRIPTION
**Pull request by antony-rheneus**
iproute2 is missing for ARM sonic-slave docker
  mcelog and firmware packages are only applicable to x86

Signed-off-by: Antony Rheneus <arheneus@marvell.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: sonic-slave-stretch/Dockerfile.j2 [hunk 1]
```diff
@@ -249,6 +249,19 @@ RUN apt-get update && apt-get install -y \
...
+        libatm1-dev \
+        libelf-dev \
+        libmnl-dev \
+        libselinux1-dev \
+        linuxdoc-tools \
+        lynx \
+        texlive-latex-extra \
+        texlive-latex-recommended \
+        iproute2 \
+{%- endif %}
```
Comment (lguohan) [inline sonic-slave-stretch/Dockerfile.j2]: there is no harm to make it for all platforms.

File: platform/marvell-armhf/docker-syncd-mrvl/Dockerfile.j2 [hunk 2]
```diff
@@ -26,7 +26,7 @@ RUN dpkg -i \
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
-COPY ["start.sh", "syncd.sh", "/usr/bin/"]
+COPY ["start.sh", "/usr/bin/"]
```
Comment (lguohan) [inline platform/marvell-armhf/docker-syncd-mrvl/Dockerfile.j2]: why remove syncd.sh?

Comment (antony-rheneus) [reply platform/marvell-armhf/docker-syncd-mrvl/Dockerfile.j2]: For Marvell right now there is no need to override common syncd.sh with platform specific syncd.sh.

File: sonic-slave-stretch/Dockerfile.j2 (hunk 1 above)
Comment (antony-rheneus) [reply sonic-slave-stretch/Dockerfile.j2]: Removed.

**Pull request merged**